src/Functions/castOrDefault.cpp

```diff
@@ -376,66 +376,475 @@ SELECT accurateCastOrDefault('abc', 'UInt32')
 
     factory.registerFunction<FunctionCastOrDefault>(accurateCastOrDefault_documentation);
 
+    FunctionDocumentation::Description toUInt8OrDefault_description = R"(
+Like [`toUInt8`](#touint8), this function converts an input value to a value of type [UInt8](../data-types/int-uint.md) but returns the default value in case of an error.
+If no `default` value is passed then `0` is returned in case of an error.
+    )";
+    FunctionDocumentation::Syntax toUInt8OrDefault_syntax = "toUInt8OrDefault(expr[, default])";
+    FunctionDocumentation::Arguments toUInt8OrDefault_arguments = {
+        {"expr", "Expression returning a number or a string representation of a number.", {"String", "Int", "Float"}},
+        {"default", "Optional. The default value to return if parsing is unsuccessful.", {"UInt8"}}
+    };
+    FunctionDocumentation::ReturnedValue toUInt8OrDefault_returned_value = {"Value of type UInt8 if successful, otherwise returns the default value if passed or 0 if not.", {"UInt8"}};
```

Comment:
```suggestion
    FunctionDocumentation::ReturnedValue toUInt8OrDefault_returned_value = {"Returns a value of type UInt8 if successful, otherwise returns the default value if passed, or 0 if not.", {"UInt8"}};
```